servers/quarkus-server/src/test/java/org/projectnessie/server/TestDisabledAuthentication.java

```diff
@@ -26,8 +26,8 @@
 import org.projectnessie.server.authn.AuthenticationDisabledProfile;
 
 /**
- * This test validates that setting `nessie.auth.enabled=false` allows all requests regardless of
- * their authentication type.
+ * This test validates that setting `nessie.authentication.enabled=false` allows all requests
```

Comment:
Should this be `nessie.server.authentication.enabled`?

Reply:
good catch, thx!

Reply:
fixed in a68aa0d0341b44dc7e39c4cf2cfa28f30dcb062e